omod/src/main/resources/config.xml

```diff
@@ -302,6 +302,14 @@
         </description>
     </globalProperty>
 
+    <globalProperty>
+        <property>coreapps.allowChangingVisitTime</property>
+        <defaultValue>true</defaultValue>
```

Comment:
Shouldn't this default to false in order to preserve the existing behaviour?

Reply:
Agreed!

Reply:
Done